### PR TITLE
Finalise docker development environment

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

File: docker/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
```
Comment (atoff) [inline docker/nginx/nginx.conf:25]: Are these comments? Do we want to get rid of them?

File: docker/nginx/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    keepalive_timeout  65;
+
+    #gzip  on;
```
Comment (atoff) [inline docker/nginx/nginx.conf:29]: As above

**Review by atoff — APPROVED** [review]

File: docker/nginx/nginx.conf (hunk 1 above)
Comment (AxonC) [reply docker/nginx/nginx.conf:25]: this is literally just pulled from the default nginx container

File: docker/nginx/nginx.conf (hunk 2 above)
Comment (AxonC) [reply docker/nginx/nginx.conf:29]: this is literally just pulled from the default nginx container

**Pull request merged**